### ISSUE TITLE
Reduced mevalonate kinase enzyme activity 

### ISSUE THREAD

**Issue by jpazmandi** [obophenotype/human-phenotype-ontology#6117]
**Preferred term label:**
Reduced mevalonate kinase enzyme activity

**Synonyms**

**Definition (free text, please give PubMed ID)**
27499643

**Parent term (use hpo.jax.org/app)**
HP:0012379

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
610377

**Your nano-attribution (ORCID)**
0000-0003-3411-9598

**Comment by pnrobinson** [comment]
added as HP_0033168
